Console/Command/QueueShell.php

```diff
@@ -11,7 +11,9 @@
 
 App::uses('Folder', 'Utility');
 App::uses('File', 'Utility');
-App::import('Vendor', 'Hipchat', array('file' => 'HipchatPhp/src/HipChat/HipChat.php')); 
+if (!class_exists('HipChat\HipChat')) {
```

Comment:
@ElThac0 if it's HipChat v2 that works properly with the namespace and the autoloader, then let's defer that for a rainy day & this still has my upthumb as is.